Add some additional examples into README (#121)
CrowdStrike/falcon-helm: helm-charts/falcon-sensor/README.md

| `container.image.repository`                     | Falcon Sensor Node registry/image name                                      | `falcon-sensor`              |
| `container.image.tag`                            | The version of the official image to use                                    | `latest`                     |
| `container.image.pullPolicy`                     | Policy for updating images                                                  | `Always`                     |
| `container.image.pullSecrets.enable`             | Enable pull secrets for private registry                                    | `false`                      |
| `container.image.pullSecrets.namespaces`         | List of Namespaces to pull the Falcon sensor from an authenticated registry | None                         |
| `container.image.pullSecrets.allNamespaces`      | Use Helm's lookup function to deploy the pull secret to all namespaces      | `false`                      |
| `container.image.pullSecrets.registryConfigJSON` | base64 encoded docker config json for the pull secret                       | None                         |

> | `container.image.sensorResources`                | The requests and limits of the sensor (see [Note below](#note-about-containerimagesensorresources))                      | None                         |

Reviewer: ```suggestion
| `container.image.sensorResources`                | The requests and limits of the sensor ([see example below](#example-using-containerimagesensorresources))                      | None                         |
```

helm show values crowdstrike/falcon-sensor
```

> #### Note about --set

Reviewer: ```suggestion
#### Note about using --set with lists
```
```suggestion
#### Note about --set
```

> #### Note about --set

Reviewer: ```suggestion
#### Note about using --set with lists
```
```suggestion
#### Note about --set
```

If you need to provide a list of values to a `--set` command, you need to escape the commas between the values e.g. `--set falcon.tags="tag1\,tag2\,tag3"`

> #### Note about container.image.sensorResources

Reviewer: ```suggestion
#### Example using container.image.sensorResources
```

> #### Note about container.image.sensorResources

Reviewer: ```suggestion
#### Example using container.image.sensorResources
```

When setting `container.image.sensorResources`, the simplest method would be to provide a values file to the `helm install` command.

Example:

```bash
helm upgrade --install falcon-helm crowdstrike/falcon-sensor \
    --set node.enabled=false
    --set container.enabled=true \
    --set falcon.cid="<CrowdStrike_CID>" \

>     --set container.image.repository="<Your_Registry>/falcon-sensor"

Reviewer: ```suggestion
    --set container.image.repository="<Your_Registry>/falcon-sensor" \
```

>     --values local.yaml

Reviewer: ```suggestion
    --values values.yaml
```
This just lines up with the helm examples even though the filename doesn't matter.

>     --values local.yaml

Reviewer: ```suggestion
    --values values.yaml
```
This just lines up with the helm examples even though the filename doesn't matter.

```

> Where `local.yaml` is

Reviewer: ```suggestion
Where `values.yaml` is
```

```yaml
container:
  sensorResources: 
    limits:
      cpu: 100m
      memory: 128Mi
    requests:
      cpu: 10m
      memory: 20Mi
```

> Of course, one could specify all options in the `local.yaml` file and skip the `--set` options altogether:

Reviewer: ```suggestion
Of course, one could specify all options in the `values.yaml` file and skip the `--set` options altogether:
```

```yaml
node:
  enabled: false
container:
  enabled: true
  image:
    repository: "<Your_Registry>/falcon-sensor"
  sensorResources: 
    limits:
      cpu: 100m
      memory: 128Mi
    requests:
      cpu: 10m
      memory: 20Mi
falcon:
  cid: "<CrowdStrike_CID>"
```

If using a local values file is not an option, you could do this:

```bash
helm upgrade --install falcon-helm crowdstrike/falcon-sensor \
    --set node.enabled=false
    --set container.enabled=true \
    --set falcon.cid="<CrowdStrike_CID>" \

>     --set container.image.repository="<Your_Registry>/falcon-sensor"
>     --set container.sensorResources.limits.memory="128Mi"
>     --set container.sensorResources.limits.cpu="100m"
>     --set container.sensorResources.requests.memory="20Mi"
>     --set container.sensorResources.requests.cpu="10m"

Reviewer: ```suggestion
    --set container.image.repository="<Your_Registry>/falcon-sensor" \
    --set container.sensorResources.limits.memory="128Mi" \
    --set container.sensorResources.limits.cpu="100m" \
    --set container.sensorResources.requests.memory="20Mi" \
    --set container.sensorResources.requests.cpu="10m"
```
